app/products/calls/connection/connection.ts

```diff
@@ -80,6 +80,9 @@ export async function newConnection(
         }
     };
 
+    // Registering WebRTC globals (e.g. RTCPeerConnection)
+    registerGlobals();
```

Comment:
older PR?

Reply:
@cpoile These are needed to make things pass because we merged some changes upstream (calls-common) but not the matching PRs.